### PR TITLE
Signup and login

### PR DESCRIPTION
**Pull request by rahulk31**
Added Signup and Login Pages.

### CODE REVIEW

File: Pages/login.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          </div>
+        </div>
+        <div class="footer-2">
+          <p class="p">Get to Know Us</p>
+          <p class="p-xs">About Us</p>
+          <p class="p-xs">Careers</p>
+          <p class="p-xs">Press Releases</p>
+          <p class="p-xs">Gift a Smile</p>
+          <p class="p-xs">Menstack Science</p>
+          <p class="p-xs">Menstack Cares</p>
```
Comment (gauravsinhaweb) [inline Pages/login.html:105]: just a minor change, use list tag for listing things.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes, Nice work!

File: Pages/login.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      </div>
+    </nav>
+
+    <!-- Hero starts -->
+    <section id="hero">
+      <span class="overlay"></span>
+      <form class="input-wrapper">
+        <div class="form-title">
+          <h5 class="h5">LOGIN</h5>
+        </div>
```
Comment (omkarpatke) [inline Pages/login.html]: suggestion - instead of writing div for h5, you can just extend the class name.

File: css/signup.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  background-color: var(--white);
+  z-index: 5;
+}
+
+.tnc-wrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
```
Comment (omkarpatke) [inline css/signup.css]: suggestion  - use px if the unit is less than 1rem.

File: Pages/login.html (hunk 1 above)
Comment (rahulk31) [reply Pages/login.html:105]: Thank you Gaurav for pointing this out. I have used this as a footer component in all pages so I will have to make changes everywhere as of now 🙊 . I will definitely change this in next update for the app.

**Review by omkarpatke — CHANGES_REQUESTED** [review]
do some minor changes which I suggested otherwise, your code looks clean and, looks good to merge.

File: Pages/login.html (hunk 2 above)
Comment (rahulk31) [reply Pages/login.html]: Done as suggested.

File: css/signup.css (hunk 3 above)
Comment (rahulk31) [reply css/signup.css]: Done as suggested.

File: Pages/login.html (hunk 2 above)
Comment (rahulk31) [reply Pages/login.html]: Thank you for pointing this out. 😃

**Pull request merged**